test/e2e/route_advertisements.go

```diff
@@ -765,6 +792,8 @@ var _ = ginkgo.DescribeTableSubtree("BGP: isolation between advertised networks"
 				asyncAssertion(func() error {
 					out, err := checkConnectivity(clientName, clientNamespace, dst)
 					if expectErr != (err != nil) {
+						framework.Logf("expected connectivity check to return error(%t), got %v, output %v", expectErr, err, out)
+						//time.Sleep(time.Hour * 1)
```

Comment:
Not needed anymore? :D

Reply:
he he yea good catch. I always leave things like this in